src/main/java/fr/xephi/authme/command/executable/email/ShowEmailCommand.java

```diff
@@ -26,9 +26,18 @@
     public void runCommand(Player player, List<String> arguments) {
         PlayerAuth auth = playerCache.getAuth(player.getName());
         if (auth != null && !Utils.isEmailEmpty(auth.getEmail())) {
-            commonService.send(player, MessageKey.EMAIL_SHOW, auth.getEmail());
+            commonService.send(player, MessageKey.EMAIL_SHOW, emailMask(auth.getEmail()));
```

Comment:
It would be good to make it configurable whether or not the email gets masked. It's not a must for now but I'm willing to bet that someone will create an issue for it in the future.